homework-no-git/06_14_2018-homework/js/app.js

```diff
@@ -0,0 +1,9 @@
+// The difference between a parameter and an argument is that parameter is a discription of what needs to be there and and argument is what is actually being calculated.
```

Comment:
you're saying basically the right thing—minor correction to your terminology—an argument is what is actually being _passed in_